src/test/java/com/ixortalk/organization/api/rest/OrganizationRestController_UsersUsed_IntegrationAndRestDocTest.java

```diff
@@ -129,6 +124,33 @@ public void asAdmin() throws JsonProcessingException {
 
     }
 
+    public void asAdminForUserWithInviteLanguageEn() throws JsonProcessingException {
+        setField(userInOrganizationXInvited, "inviteLanguage", "en");
+        userRestResource.save(userInOrganizationXInvited);
+
+        given()
+                .auth()
+                .preemptive()
+                .oauth2(USER_IN_ORGANIZATION_X_ADMIN_ROLE_JWT_TOKEN)
+                .filter(
+                        document("organizations/users-used/ok",
+                                preprocessRequest(staticUris(), prettyPrint()),
+                                preprocessResponse(prettyPrint()),
+                                requestHeaders(TOKEN_WITH_ORGANIZATION_ADMIN_PRIVILEGES),
+                                pathParameters(ORGANIZATION_ID_PATH_PARAMETER),
+                                requestFields(
+                                        fieldWithPath("[]").description("A list of logins that have been used")
+                                )
+                        ))
+                .contentType(JSON)
+                .body(objectMapper.writeValueAsString(newArrayList(userInOrganizationXCreated.getLogin(), userInOrganizationXInvited.getLogin())))
+                .post("/organizations/{id}/users/used", organizationX.getId())
+                .then()
+                .statusCode(SC_OK);
+
+        mailingServiceWireMockRule.verify(1, postRequestedFor(urlEqualTo("/mailing/send")).withRequestBody(containing("\"languageTag\":\"nl\"")));
```

Comment:
This is not correctly tested, `inviteLanguage` gets set to `en` explicitly in this test but the verify happens with `nl`?